src/Microsoft.Kusto.ServiceLayer/QueryExecution/Query.cs

```diff
@@ -333,32 +331,35 @@ public Task<ExecutionPlan> GetExecutionPlan(int batchIndex, int resultSetIndex)
         /// </summary>
         private async Task ExecuteInternal()
         {
-            ReliableDataSourceConnection sqlConn = null;
+            ReliableDataSourceConnection queryConnection = null;
```

Comment:
Renamed to queryConnection since sqlConn was never used. Guessing this was the original intention.